tools/pwsh/clone_herbert_branch.ps1

```diff
@@ -16,12 +16,14 @@ param(
 $HERBERT_URL = "https://github.com/pace-neutrons/Herbert.git"
 $HERBERT_DIR = "$($(pwd).Path)/Herbert"
 
-Write-Output "Building Herbert branch $branch..."
+Write-Output "Building Herbert branch '$branch'..."
 if (Test-Path -Path "$HERBERT_DIR") {
   Write-And-Invoke "git -C $HERBERT_DIR fetch origin"
   Write-And-Invoke "git -C $HERBERT_DIR reset --hard origin/$branch"
 } else {
   Write-And-Invoke "git clone $HERBERT_URL --depth 1 --branch $branch $HERBERT_DIR"
 }
-$build_cmd = "$HERBERT_DIR/tools/build_config/build.ps1 -build -build_tests OFF $build_args"
+
+$build_cmd = "$HERBERT_DIR/tools/build_config/build.ps1 -build"
+$build_cmd += " -build_tests OFF $build_args"
```

Comment:
it would be clearer to split this over two lines; make the `build_test OFF` explicit